docs/running-node-aws-terraform-ansible/deployment.md

```diff
@@ -0,0 +1,184 @@
+    
+Create credentials file [`~/.aws/credentials`] with the following content:
+```text
+[default]
+aws_access_key_id = <access_key_id_here>
+aws_secret_access_key = <secret_access_key_here>
+```
+> **_Note:_** Your account must have enough privileges to manage all AWS resources required by terraform
+
+### 2. Disable host key checking for Ansible (to avoid host key checking when Ansible connects to AWS instances using ssh)
+
+Create Ansible configuration file [`~/.ansible.cfg`] with the following content:
+```
+ANSIBLE_HOST_KEY_CHECKING=False
+```
+
+## Deployment Configuration
+
+### 1. Configure AWS infrastructure parameters in [`deployment/terraform/aws/terraform.tfvars`]
+    
+### AWS Regions:
+```
+region_1 = "us-west-1"
+region_2 = "us-east-2"
+```
+- Selects two regions where nodes will be created
+
+### (Genesis) Validator:
+```
+validator_config = {
+    instance_type = "t3.medium"
+}
+```
+- Validator node is created in `region_1` by default
+
+
+### Private Sentries (optional):
+
+```hcl
+private_sentries_config = {
+    enable        = true
+    nodes_count   = 2
+    instance_type = "t3.medium"
+}
+```
+- Private sentry nodes are created in the region as Validator by default
+- Can be disabled by setting `enable = false`
+- Only one instance of private sentry is created with static ip address
+
+### Public Sentries (optional):
+```hcl
+public_sentries_config = {
+    enable        = true
+    enable_ipv6   = false
+    nodes_count   = 2
+    instance_type = "t3.medium"
+
+    regions = [
+        1,
+        2
+    ]
+}
+```
+- Requires `Private Sentries` being enabled
+- Can be configured to have IPv6 addresses using `enable_ipv6 = true`
+    > **_Note:_** Number of available IPv4 static addresses is restricted to 5 per region on AWS by default
+- Can be configured to run on multiple regions
+
+### Observers (optional):
+```hcl
+observers_config = {
+    enable           = true
+    nodes_count      = 3
+    instance_type    = "t3.medium"
+    root_domain_name = "matterprotocol.com"
+    enable_tls       = true
+
+    regions = [
+        1,
+        2
+    ]
+}
+```
+
+- Requires `Private Sentries` being enabled
+- When `root_domain_name` parameter is set, will be serving requests under `on.<root_domain_name>` domain
+    - Name servers for `root_domain_name` servers must be pointed to AWS and managed by the AWS account
+- TLS can be enabled or disabled
+- Can be configured to run on multiple regions
+
+### Prometheus (optional)
+```hcl
+prometheus_config = {
+  enable        = true
+  instance_type = "t3.small"
+}
+```
+
+- Requires `Private Sentries` being enabled
+- When enabled runs a dedicated Prometheus server on Private Sentries VPC to collect Tendermint metrics from all DCL nodes
+- Collected metrics are written to AWS [AMP workspace](https://aws.amazon.com/prometheus/)
+
+### 2. Configure Ansible inventory variables in [`deployment/ansible/inventory/aws/group_vars/all.yaml`]
+```yaml
+chain_id: test-net2
```

Comment:
Please explicitly specify the values for the current DCL Test Net in addition to the abstract parameters.